src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -188,15 +188,19 @@ private void checkPersonName(Name name, String personType, Appellant appellant)
 
     private void checkPersonAddressAndDob(Address address, Identity identity, String personType, Map<String, Object> caseData, Appellant appellant) {
 
+        boolean isAddressLine4Present = doesAddressLine2Exist(address);
```

Comment:
Is this correct? Method looks for address line 2 but variable says address line 4

Reply:
ah typo..